### PR TITLE
Initial check-in of script for deleting dataset entirely from spandex secondary

### PR DESCRIPTION
**Pull request by rlvoyer**
This new script will ensure that the dataset is removed from Spandex as well as from the secondary
manifest via datacoordinator.

### CODE REVIEW

File: bin/snapshot_recovery_progress.sh [hunk 1]
```diff
@@ -20,7 +18,7 @@ esac
 
 INDEX=$2
 if [ "$CLUSTER" == "" -o "$INDEX" == "" ]; then
-  echo "Usage $0 <local|staging|rc|rpod|eu-prod|fedramp-prod> <index_name_or_alias>"
```
Comment (kavedder) [inline bin/snapshot_recovery_progress.sh]: Hahahahahahahaha

File: bin/delete_dataset_from_secondary.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    DATACOORDINATOR="http://data-coordinator.app.aws-us-west-2-staging.socrata.net"
+    CLUSTER="http://spandex.elasticsearch.aws-us-west-2-staging.socrata.net" ;;
+  "rc" )
+    DATACOORDINATOR="http://data-coordinator.app.aws-us-west-2-rc.socrata.net"
+    CLUSTER="http://spandex.elasticsearch.aws-us-west-2-rc.socrata.net" ;;
+  "eu-prod" )
+    DATACOORDINATOR="http://data-coordinator.app.aws-us-west-2-rc.socrata.net"
+    CLUSTER="http://spandex.elasticsearch.aws-eu-west-1-prod.socrata.net" ;;
+  "fedramp-prod" )
+    CLUSTER="http://spandex-6.elasticsearch.aws-us-east-1-fedramp-prod.socrata.net" ;;
```
Comment (kavedder) [inline bin/delete_dataset_from_secondary.sh]: what is `$DATACOORDINATOR` in the case of fedramp-prod?

File: bin/delete_dataset_from_secondary.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+STACK=$1
+DCU=alpha # data coordinator universe or whatever
```
Comment (kavedder) [inline bin/delete_dataset_from_secondary.sh]: Looks like you don't use DCU in this script (and at line 9)

Comment (rlvoyer) [inline bin/delete_dataset_from_secondary.sh]: Thanks. I had made changes on the bastion that I forgot to check in.

**Pull request merged**